crawl-ref/source/map-knowledge.h

```diff
@@ -41,3 +42,10 @@ map_feature get_cell_map_feature(const map_cell& cell);
 bool is_explore_horizon(const coord_def& c);
 
 void reautomap_level();
+
+/**
+ * @brief Get the bounding box of the known map.
+ *
+ * @return vector of [topleft coord, bottomright coord] of bbox.
```

Comment:
This should say pair, not vector (and use curly braces probably?)

Reply:
done